packages/calypso-products/src/is-credits.ts

```diff
@@ -1,3 +1,6 @@
-export function isCredits( product: { product_slug: string } ): boolean {
-	return 'wordpress-com-credits' === product.product_slug;
+import { camelOrSnakeSlug } from './camel-or-snake-slug';
+import type { WithSnakeCaseSlug, WithCamelCaseSlug } from './types';
```

Comment:
Fixed the ordering!

I definitely can from relative to absolute, but the imports in the other files in the same folder are all relative FYI.

Reply:
As the imports are new, I think we should add them as absolute imports :) It's fine to keep other files unrelated to this change as they are, someday if we have to revisit those files, we can modify them or even do a specially designed task for that purpose.

Reply:
I took a stab at this, but ran into trouble. Turns out that "self-referencing" the package by doing something like `import { camelOrSnakeSlug } from '@automattic/calypso-products'` results in problems, at least for the test suite (the import doesn't resolve, even though it's supposed to).

Since there is an existing convention of relative imports in this package and since we can't easily change to absolute imports, we decided to keep using relative imports.